curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-153-investigating-gaussian-integers.md

```diff
@@ -8,59 +8,73 @@ dashedName: problem-153-investigating-gaussian-integers
 
-The regular integers are also Gaussian integers (with b=0).
+The regular integers are also Gaussian integers (with $b = 0$).
 
-To distinguish them from Gaussian integers with b ≠ 0 we call such integers "rational integers."
+To distinguish them from Gaussian integers with $b ≠ 0$ we call such integers "rational integers."
 
 A Gaussian integer is called a divisor of a rational integer n if the result is also a Gaussian integer.
 
-If for example we divide 5 by 1+2i we can simplify in the following manner:
+If for example we divide 5 by $1 + 2i$ we can simplify in the following manner:
 
-Multiply numerator and denominator by the complex conjugate of 1+2i: 1−2i.
+Multiply numerator and denominator by the complex conjugate of $1 + 2i$: $1 − 2i$.
 
-The result is .
+The result is:
 
-So 1+2i is a divisor of 5.
+$$\frac{5}{1 + 2i} = \frac{5}{1 + 2i} \frac{1 - 2i}{1 - 2i} = \frac{5(1 - 2i)}{1 - {(2i)}^2} = \frac{5(1 - 2i)}{1 - (-4)} = \frac{5(1 - 2i)}{5} = 1 - 2i$$
 
-Note that 1+i is not a divisor of 5 because .
+So $1 + 2i$ is a divisor of 5.
 
-Note also that if the Gaussian Integer (a+bi) is a divisor of a rational integer n, then its complex conjugate (a−bi) is also a divisor of n. In fact, 5 has six divisors such that the real part is positive: {1, 1 + 2i, 1 − 2i, 2 + i, 2 − i, 5}.
+Note that $1 + i$ is not a divisor of 5 because:
+
+$$\frac{5}{1 + i} = \frac{5}{2} - \frac{5}{2}i$$
+
+Note also that if the Gaussian Integer ($a + bi$) is a divisor of a rational integer $n$, then its complex conjugate ($a − bi$) is also a divisor of $n$. In fact, 5 has six divisors such that the real part is positive: {1, 1 + 2i, 1 − 2i, 2 + i, 2 − i, 5}.
 
 The following is a table of all of the divisors for the first five positive rational integers:
 
-n Gaussian integer divisors with positive real partSum s(n) of these
+| n | Gaussian integer divisors with positive real part | Sum s(n) of these divisors |
+|---|---------------------------------------------------|----------------------------|
+| 1 | 1                                                 | 1                          |
+| 2 | 1, 1 + i, 1 - i, 2                                | 5                          |
+| 3 | 1, 3                                              | 4                          |
+| 4 | 1, 1 + i, 1 - i, 2, 2 + 2i, 2 - 2i, 4             | 13                         |
+| 5 | 1, 1 + 2i, 1 - 2i, 2 + i, 2 - i, 5                | 12                         |
+
+For divisors with positive real parts, then, we have: $\displaystyle\sum_{n=1}^5 s(n) = 35$.
+
+For $1 ≤ n ≤ {10}^5$, $\displaystyle\sum_{n = 1}^{{10}^5} s(n) = 17924657155$.
 
-divisors111 21, 1+i, 1-i, 25 31, 34 41, 1+i, 1-i, 2, 2+2i, 2-2i,413 51, 1+2i, 1-2i, 2+i, 2-i, 512 For divisors with positive real parts, then, we have: . For 1 ≤ n ≤ 105, ∑ s(n)=17924657155. What is ∑ s(n) for 1 ≤ n ≤ 108?
+What is $\displaystyle\sum s(n)$ for $1 ≤ n ≤ {10}^8$?
```

Comment:
This one doesn't look the same on [euler](https://projecteuler.net/problem=153):
![Screen Shot 2021-07-09 at 8 02 30 AM](https://user-images.githubusercontent.com/20648924/125081810-0cd10400-e08c-11eb-91a2-162386d98174.png)
This PR:
![Screen Shot 2021-07-09 at 8 02 47 AM](https://user-images.githubusercontent.com/20648924/125081834-165a6c00-e08c-11eb-81c6-c17fbbfd8dee.png)

Not sure if this matters. Maybe not a problem.